Source/WebCore/page/SecurityOriginData.h

```diff
@@ -76,11 +89,11 @@ struct SecurityOriginData {
 
     bool isNull() const
     {
-        return protocol.isNull() && host.isNull() && port == std::nullopt;
+        return protocol.isNull() && host.isNull() && port == std::nullopt && !opaqueOriginIdentifier;
```

Comment:
We shouldn't need this change since protocol/host wouldn't be null if opaqueOriginIdentifier was non-null.